lib/que/sql.rb

```diff
@@ -50,9 +50,10 @@ module Que
         FROM (
           SELECT j
           FROM que_jobs AS j
-          WHERE queue = $1::text
+          WHERE (queue = $1::text OR queue = '')
```

Comment:
Just for my own understanding, why this change?

Reply:
We want to run two sets of workers, one consuming jobs from a `gcp` queue and another from `softlayer`. We also want both of these queues to consume work from the default queue, as we won't be changing all jobs to be one queue or the other, we'll be using priority threshold to gradually permit GCP workers to take more jobs.

This should be documented [here](https://paper.dropbox.com/doc/Que-Migration-Dependencies-per-env-bBt1eYG1j3OZVhMIQF64U#:uid=061592766615478911359257&h2=Configuration)

Reply:
Thanks for linking the doc, I had missed it!